oval_graph/command_line_client/client.py

```diff
@@ -1,93 +1,95 @@
-import re
 import argparse
-import tempfile
-import os
-import webbrowser
-import subprocess
-from datetime import datetime
+import re
 import sys
+from datetime import datetime
 
-from .xml_parser import XmlParser
-from .exceptions import NotChecked
-from .__init__ import __version__
+from ..__init__ import __version__
```

Comment:
` from .. import __version__` should work as well - contents of `__init__.py` become part of the respective python *package*